pkg/k8s/namespaces/namespace.go

```diff
@@ -349,3 +350,14 @@ func (t *Trip) listAll(ctx context.Context, traveler Traveler, client dynamic.Re
 		return nil
 	})
 }
+
+// isStorage returns if the kind is some of storage kind
+func isStorage(kind string) bool {
+	switch kind {
```

Comment:
NIT: Have you thought about using the switch case with multiple values?
```suggestion
	switch kind {
	case volumeKind, volumeSnapshotKind:
	    return true
	}
```

Reply:
I though about it, in cases where there aren't too many branches I prefer to have them explicitly split them individually, but not strong opinion, I'll change it